profiler/src/ProfilerEngine/Datadog.Profiler.Native/Sample.h

```diff
@@ -56,10 +56,11 @@ static constexpr size_t array_size = sizeof(SampleTypeDefinitions) / sizeof(Samp
 //---------------------------------------------------------------
 
 typedef std::array<int64_t, array_size> Values;
-typedef std::pair<std::string, std::string> Label; // TODO: use stringview to avoid copy
+typedef std::pair<std::string_view, std::string> Label; // TODO: use stringview to avoid copy
```

Comment:
Remove the comment since you are using stringview for the label text (keeping  string for its value seems reasonable)